src/index.js

```
@@ -3,7 +3,16 @@
 'use strict';
 
 const { createServer } = require('./createServer');
+const { sequelize } = require('./db');
 
-createServer().listen(5700, () => {
-  console.log('Server is running on localhost:5700');
-});
+async function start() {
+  try {
+    await sequelize.sync();
+    console.log('✅ Database synchronized');
+
+    createServer().listen(5700, () => {
+      console.log('Server is running on localhost:5700');
+    });
+  } catch {}
```

This empty `catch` block swallows any errors that happen during startup, like a database connection failure. The requirements state that you should log the error (e.g., using `console.error`) and then exit the process with a non-zero status code to ensure failures are visible.